src/PhotoRail.h

```diff
@@ -0,0 +1,44 @@
+#ifndef PHOTO_RAIL_H
+#define PHOTO_RAIL_H
+
+#include <Arduino.h>
+
+#include "IrSony.h"
+#include "Rail.h"
+
+class PhotoRail
+ : public Rail
+{
+private:
+    int left = 0;
+    int right = 0;
+
+    IrSony ir_sony;
```

Comment:
We should abstract this to interface; sth like:
`class CameraRemote { virtual void shoot(); }`

Reply:
sounds good! I will try it.

Reply:
or better: let us do it after the merge